amt/core/db.py

```diff
@@ -11,18 +9,13 @@
 logger = logging.getLogger(__name__)
 
 
-@lru_cache(maxsize=8)
 def get_engine() -> Engine:
     settings = get_settings()
-    connect_args = (
-        {"check_same_thread": False, "isolation_level": None} if settings.APP_DATABASE_SCHEME == "sqlite" else {}
-    )
-    poolclass = StaticPool if settings.APP_DATABASE_SCHEME == "sqlite" else QueuePool
+    connect_args = {"check_same_thread": False} if settings.APP_DATABASE_SCHEME == "sqlite" else {}
 
     return create_engine(
         settings.SQLALCHEMY_DATABASE_URI,  # pyright: ignore [reportArgumentType]
         connect_args=connect_args,
-        poolclass=poolclass,
```

Comment:
Can this be removed because we have unique databases now? Or because the E2E tests can not insert things anymore?

Reply:
It was only needed for memory databases